### ISSUE TITLE
Invalid floating point application

### ISSUE THREAD

**Issue by grahamegrieve** [LongDirtyAnimAlf/fpcupdeluxe#583]
Trying to use fpcdeluxe-aarch64-linux on a Mac M1 under parallels:

![image](https://github.com/LongDirtyAnimAlf/fpcupdeluxe/assets/5858581/aaf7cd12-2a4e-4d39-8a54-900a410af3b4)

This happens at start up, or install/update, and doesn't seem to depend on which FPC version or Lazarus version is selected

**Comment by LongDirtyAnimAlf** [comment]
I do not own a Mac. I have only a AMD64 VM running OSX. That will limit the amount of help I can give.
However. What is parallels ?

**Comment by grahamegrieve** [comment]
yes, I'm not surprised this'll be a challenge. Parallels is a Mac VM - see https://www.parallels.com. The support on M1 Macs is incomplete and various random things crash. 

I'll have to install Lazarus without fpcpdeluxe if I want to investigate, right? What do I need to install? But honestly, the problem is likely in the fps library pretty deep somewhere

**Comment by LongDirtyAnimAlf** [comment]
Well.
Fpupdeluxe runs when started. That might mean that the problem might be located inside fpcupdeluxe, within its functionality, when used. But, as far as I know, up does not use floating point math.

**Comment by LongDirtyAnimAlf** [comment]
Would you mind trying the command line version : fpcup itself ?
https://github.com/LongDirtyAnimAlf/Reiniero-fpcup/releases/tag/v2.2.0q
